alibi/explainers/tests/test_partial_dependence.py

```diff
@@ -281,36 +297,52 @@ def test_grid_points_error(adult_data, use_int):
         explainer._grid_points_sanity_checks(grid_points, n_features=X_train.shape[1])
 
 
-@pytest.mark.parametrize('n_ice', ['all', 'list', 'int'])
-@pytest.mark.parametrize('n_samples, n_values', [(100, 10)])
-def test_ice_sampling(n_ice, n_samples, n_values):
-    """ Checks if the ice sampling helper function works properly when the arguments are valid. """
-    ice_vals = np.random.randn(n_values, n_samples)
-
-    if n_ice == 'all':
-        ice_sampled_vals = _sample_ice(ice_values=ice_vals, n_ice=n_ice)
-        np.testing.assert_allclose(ice_vals, ice_sampled_vals)
-
-    elif n_ice == 'list':
-        size = np.random.randint(1, n_samples)
-        # needs to be sorted because of the np.unique applied inside the _sample_ice
-        n_ice = np.sort(np.random.choice(n_samples, size=size, replace=False)).tolist()
-        ice_sampled_vals = _sample_ice(ice_values=ice_vals, n_ice=n_ice)
-        np.testing.assert_allclose(ice_vals[:, n_ice], ice_sampled_vals)
-
+def compare_feature_values(exp_alibi: Explanation, exp_sklearn: Bunch):
```

Comment:
I would name this starting with `assert` (e.g. `assert_feature_values_equal`) similar to `numpy.testing` utility functions as it makes clear to the developer that the function performs assertions.